features/eolearn/tests/test_interpolation.py

```diff
@@ -230,41 +230,47 @@ def execute(self, eopatch):
 
 
 COPY_FEATURE_CASES = [
-    InterpolationTestCase(
-        "cubic_copy_success",
-        CubicInterpolationTask(
-            (FeatureType.DATA, "NDVI"),
-            result_interval=(0.0, 1.0),
-            mask_feature=(FeatureType.MASK, "IS_VALID"),
-            resample_range=("2015-01-01", "2018-01-01", 16),
-            unknown_value=5,
-            bounds_error=False,
-            copy_features=[
-                (FeatureType.MASK, "IS_VALID"),
-                (FeatureType.DATA, "NDVI", "NDVI_OLD"),
-                (FeatureType.MASK_TIMELESS, "LULC"),
-            ],
+    (
+        InterpolationTestCase(
+            "cubic_copy_success",
+            CubicInterpolationTask(
+                (FeatureType.DATA, "NDVI"),
+                result_interval=(0.0, 1.0),
+                mask_feature=(FeatureType.MASK, "IS_VALID"),
+                resample_range=("2015-01-01", "2018-01-01", 16),
+                unknown_value=5,
+                bounds_error=False,
+                copy_features=[
+                    (FeatureType.MASK, "IS_VALID"),
+                    (FeatureType.DATA, "NDVI", "NDVI_OLD"),
+                    (FeatureType.MASK_TIMELESS, "LULC"),
+                ],
+            ),
+            result_len=69,
+            expected_statistics={},
```

Comment:
this probably has to do with changes to tests, but where are the values to compare with now?

Reply:
these test cases are only used to check that the `copy_features` parameter works as intended. Even the old `test_copied_fields` (where these cases are used) never checked the statistics at all.